### PR TITLE
fix: install and remove phpdox as required

### PR DESCRIPTION
**Pull request by childish-sambino**
This is a patch to unblock other dependency work.

### CODE REVIEW

File: tests/phpunit.xml [hunk 1]
```diff
@@ -8,8 +8,5 @@
     <testsuite name="Twilio Integration Test Suite">
       <directory>./Twilio/Integration</directory>
     </testsuite>
-    <testsuite name="Twilio Services Test Suite">
```
Comment (shwetha-manvinkurke) [inline tests/phpunit.xml:11]: why was this removed?

**Review by shwetha-manvinkurke — APPROVED** [review]

File: tests/phpunit.xml (hunk 1 above)
Comment (childish-sambino) [reply tests/phpunit.xml:11]: It doesn't actually exist in the repo. Removing phpdox from the dependencies list caused phpunit to use a newer version (since phpdox was lowering it before) which caused the tests to break because of this suite.
